---
sensor_msgs/Image last_image
